packages/mobile/src/identity/contactMapping.ts

```diff
@@ -200,10 +201,7 @@ export function* fetchAddressesAndValidateSaga({
       e164Number
     )
 
-    if (addressValidationType !== AddressValidationType.NONE) {
-      yield put(requireSecureSend(e164Number, addressValidationType))
-    }
-
+    yield put(requireSecureSend(e164Number, addressValidationType))
```

Comment:
Why did should the condition be removed? I think without this we will unnecessarily be forcing users through Secure Send

Reply:
In this case, we check all the time if it is necessary to use the Secure Send. The problem was that sometimes the check says that is not needed the verification and it is not stored in the state, so the user went to Secure Send.

Reply:
In this case we check all the time if it is necessary to use the Secure Send. The problem was that sometimes the check says that is not needed the verification and it is not stored in the state, so the user went to Secure Send.

Reply:
I'm confused - can you give me an example when this will break?

The issue I see with removing it is that users will have to keep doing Secure Send even after they validate the recipients address. To test this, try sending to someone with multiple addresses and go through Secure Send. Then after, try to send to that person again...I think you will have to do Secure Send again because `requireSecureSend` sets `address = undefined` in the reducer